modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/gcal/GcalController.scala

```diff
@@ -23,50 +21,59 @@ trait GcalController {
 
 object GcalController {
 
-  trait LampState
+  sealed trait LampState extends Product with Serializable
 
   object LampState {
 
     case object Off extends LampState
 
     case object On extends LampState
 
+    implicit val eq: Eq[LampState] =
+      Eq.fromUniversalEquals
+
   }
 
-  final case class ArLampState(self: LampState) extends AnyVal
```

Comment:
Why did you remove the `extends AnyVal` ?

Reply:
It doesn't benefit you too much with a case class